src/devices/SenseHat/SenseHat.cs

```diff
@@ -53,96 +83,96 @@ public SenseHat(I2cBus? i2cBus = null, bool shouldDispose = false)
         /// Fills LED matrix with specified color
         /// </summary>
         /// <param name="color">Color to fill LED matrix with</param>
-        public void Fill(Color color = default(Color)) => _ledMatrix.Fill(color);
+        public void Fill(Color color = default(Color)) => LedMatrix.Fill(color);
```

Comment:
Can those type of functions be `[Command]` ?

Reply:
I wasn't sure about annotating any of the displays yet so I left them out of scope for now.

Reply:
Fair, let's see first with the first sensors. Impatient to see all this coming together :-)